SECURITY.md

```diff
@@ -0,0 +1,41 @@
+# Security Policy
+
+At ZITADEL we are extremely grateful for security aware people that disclose vulnerabilities to us and the open source community. All reports will be investigated by our team.
+
+## Supported Versions
+
+After the initial Release the following version support will apply
+
+| Version | Supported                               |
+| ------- | ------------------                      |
+| 0.x.x   | :white_check_mark:                      |
```

Comment:
```suggestion
| 0.x.x   | :white_check_mark:  |
| 1.x.x   | :white_check_mark:  |
| 2.x.x   | :white_check_mark:  |
```

Reply:
0.x / 1.0 are the same module, 2.0 is currently an alpha release, but will be changed with https://github.com/zitadel/zitadel-go/pull/143